app/config/urls/challenge_subdomain.py

```diff
@@ -15,6 +15,10 @@
         ),
         name="subdomain_robots_txt",
     ),
+    path(
+        "components/",
+        include("grandchallenge.components.urls", namespace="components"),
+    ),
```

Comment:
The interface create form links to the component overview pages. In order to make that possible for the phase views, the subdomain url config needs to be adjusted, or so I think. Is there a better way of solving this?

Reply:
I think this is the correct thing to do - this is why we have `markdownx/` here.